src/ctapipe/containers.py

```diff
@@ -166,6 +167,15 @@ class EventType(enum.Enum):
     UNKNOWN = 255
 
 
+class VarianceType(enum.Enum):
+    """Enum of variance types used for the DL1PedestalVarianceContainer"""
+
+    # Simple variance of waveform
+    SIMPLE = 0
+    # Variance of integrated samples of a waveform
+    SAMPLE = 1
```

Comment:
SIMPLE and SAMPLE are extremely similar and `SAMPLE` could also mean "variance over the waveform samples" so what you refer to here as "SIMPLE".

Maybe better `SAMPLE` vs. `INTEGRATED` ?

Also please add a color before the comment: `#: ` which is the special syntax for docstrings for member attributes.